test/normalize.html

```diff
@@ -0,0 +1,493 @@
+<!DOCTYPE html>
```

Comment:
Yeah, I saw the tests you added and those look good. I'm talking about the file that we're commenting on: isn't this a copy of https://github.com/necolas/normalize.css/blob/master/test.html (one of Normalize's test files) and therefore unnecessary?